system/CLI/GeneratorTrait.php

```diff
@@ -212,17 +212,20 @@ protected function qualifyClassName(): string
 		{
 			// @codeCoverageIgnoreStart
 			$nameLang = $this->classNameLang ?: 'CLI.generator.className.default';
```

Comment:
why **CLI.generator.className.default** !! Do you mean  **CLI.generator.className** ??

Reply:
No. The generator class names have been modified to be specific to the component they are creating. For example, in `make:model` if you are missing the class input you will be prompted with `Model class name:` instead of the generic `Class name`. `CLI.generator.className.default` is the lang string for the `Class name` input.

Reply:
Ok.
Do not need to create a string className.default in the system \ Language \ en \CLI.php file?